frontend/src/views/CellGuide/components/common/OntologyDagView/index.tsx

```diff
@@ -481,13 +500,21 @@ function setCellCountsInRawTree(
 ) {
   const cellTypeId = graph.id.split("__").at(0) ?? graph.id;
 
-  graph.n_cells = tissueCounts[cellTypeId]?.n_cells ?? graph.n_cells;
-  graph.n_cells_rollup =
-    tissueCounts[cellTypeId]?.n_cells_rollup ?? graph.n_cells_rollup;
+  graph.n_cells = tissueCounts[cellTypeId]?.n_cells ?? 0;
```

Comment:
this change makes it so that cell types that aren't present in the tissue will be removed.

Reply:
maybe add a comment to explain the intention!